### PR TITLE
Update dashboards to use datastream.dataset instead of dataset.name

### PR DESCRIPTION
**Pull request by ruflin**
WIP

### CODE REVIEW

File: packages/apache/kibana/search/Apache-HTTPD-ecs.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
                 "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.index",
                 "query": {
                     "language": "kuery",
-                    "query": "(dataset.name:apache.status)"
+                    "query": "(datastream.dataset:apache.status)"
```
Comment (ycombinator) [inline packages/apache/kibana/search/Apache-HTTPD-ecs.json:27]: Do we need to worry about making this backwards compatible, like:

```
(datastream.dataset:apache.status or dataset.name:apache.status)
```

Comment (ruflin) [reply packages/apache/kibana/search/Apache-HTTPD-ecs.json:27]: No, as the agent ships currently both fields with the same values.

**Comment by mtojek** [comment]
If we want to release them, we need to bump up versions.

**Comment by ycombinator** [comment]
> If we want to release them, we need to bump up versions.

Good point. Do we bump up versions now (i.e. with every change to a package) or right before release (i.e. moving to the `package-storage` repo so a bunch of changes are grouped together under a single version bump)?

**Comment by ruflin** [comment]
@mtojek @ycombinator As I forsee a few more changes coming in the next 24h, lets bump separately and get all changes in first.

**Pull request merged**